runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -656,11 +656,6 @@ void __quantum__qis__Probs(MemRefT_double_1d *result, int64_t numQubits, ...)
 
     const size_t numElements = 1U << numQubits;
 
-    if (numElements != sv_probs.size()) {
```

Comment:
Was this check redundant?

Reply:
Yep. We check these in the Simulator implementation.